adapter/config/crd/conversion.go

```diff
@@ -45,7 +45,7 @@ func convertObject(schema model.ProtoSchema, object IstioObject, domain string)
 
 // convertConfig translates Istio config to k8s config JSON
 func convertConfig(schema model.ProtoSchema, config model.Config) (IstioObject, error) {
-	spec, err := schema.ToJSONMap(config.Spec)
+	spec, err := model.ToJSONMap(config.Spec)
```

Comment:
Renaming/refactoring should be done in a separate PR with no functional changes.